js/tests/unit/collapse.js

```diff
@@ -177,6 +177,34 @@ $(function () {
     setTimeout(done, 0)
   })
 
+  QUnit.test('should not show a collapse when initialized with "hide" if already hidden', function (assert) {
+    assert.expect(0)
+    var done = assert.async()
+
+    $('<div class="collapse"></div>')
+      .appendTo('#qunit-fixture')
+      .on('show.bs.collapse', function () {
+        assert.ok(false, 'showing a previously-uninitialized hidden collapse when "hide" method called')
```

Comment:
Again, not a native speak but shouldn't it be "showing a previously-uninitialized hidden collapse when "hide" method *is* called"